src/rust/inetstack/protocols/udp/mod.rs

```diff
@@ -19,6 +19,9 @@ mod tests;
 //==============================================================================
 
 pub use self::{
-    datagram::UdpHeader,
+    datagram::{
+        UdpDatagram,
```

Comment:
Is this required?

Reply:
Yes, it is. The UDP datagram should be exposed to the network simulator, as it should be able to create and parse UDP datagrams. Note that TCP segment is also exposed. But good idea, I can investigate if we can expose it only on test builds.